### DISCUSSION TITLE
Due to PowerShell always run as administrator, I cannot install ohmyposh

### DISCUSSION THREAD

**Discussion by Ran-oops** [JanDeDobbeleer/oh-my-posh#2437] [Troubleshoot] [answered]
Upon restarting powershell, ohmyposh will not work. So, how should I disable `run as admin default`?

# FYI
Windows Terminal **default** config: `run as admin` is disabled
Windows Terminal **PowerShell** config: `run as admin` is disabled
Powershell>Properties>Advanced: `run as admin` is disabled

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Ran-oops I don't think I understand the issue? You can't install for your local user as WT is always admin? You can always download the installer from the releases here as well (it's the same).

**Reply by Ran-oops** [reply]
Hi, @JanDeDobbeleer 
- install oh-my-posh by CLi is `OK`
- reopen a new tab is `NOT OK`

- since the powershell always run as admin, I guess if the admin mode is causing this issue?
- 
![image](https://user-images.githubusercontent.com/58836736/175754282-05a6c43d-04be-430e-a721-60fa0dee9c7c.png)

**Reply by lewis-yeung** [reply]
@Ran-oops Nothing to do with whether it is run as admin or not. You should add the init command line in your PowerShell profile. See [docs](https://ohmyposh.dev/docs/installation/prompt).

**Reply by Ran-oops** [reply]
> @Ran-oops Nothing to do with whether it is run as admin or not. You should add the init command line in your PowerShell profile. See [docs](https://ohmyposh.dev/docs/installation/prompt).

Thanks :)
